CHANGELOG.rst

```diff
@@ -2,9 +2,15 @@
 What's New
 ==========
 
+=======
 climpred v2.1.x (2021-0x-xx)
 ============================
 
+Breaking changes
+----------------
+- `~climpred.classes.PrefectModelEnsemble.verify` does not add climpred
```

Comment:
Typo in `PerfectModelEnsemble`

Reply:
I don't think this is a "breaking change". Breaking change means that if you have existing code it won't work anymore. This seems like more aesthetics to the package than a breaking change.

Reply:
ok. then internal